examples/rn-mbnb/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "frontend",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@realm/react": "^0.6.2",
```

It'd be great to push the package lock changes after installing with this version as well 🙂 
```suggestion
    "@realm/react": "^0.7.0",
```